index.html

```diff
@@ -3,9 +3,195 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width initial scale=1.0">
-        <title></title>
-        <link rel="stylesheet" href="./assets/styles/style.css">
+        <title>Ai Ally</title>
+        <link rel="stylesheet" href="../app-landing-page/assets/styles/style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
     </head>
     <body>
+        <header>
+            <div class="header-line">
+                <div class="first-section">
+                <img id="ai-ally-log" width="10hw" height="10vh" src="./assets/images/ai ally logo.png" alt="ai-ally-logo">Ai Ally
+                </div>
+                <div class="second-section">
+                <div id="header-line-home">Home</div>
+                <div id="header-line-features">Features</div>
+                <div id="header-line-pricing">Pricing</div>
+                <div id="header-line-testimonials">Testimonials</div>
+                </div>
+                <div class="third-section">
+                <div id="header-line-sign-in">Sign in</div>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="black-background">
+                <div class="text">
+                <div id="black-background-text"><h1>Unleash the full potential of AI</h1> AI Ally is a versatile asssistant that
+                     utilizes state-of-the-art matural language processing to prove real-time support and assistance across various
+                     domains.
+                </div>
+                <div class="black-background-text2">
+                  <div id="lime-green-button">Get the App &rarr; </div>
+                  <div>more</div>
+                </div>
+                <div class="black-background-text3">
+                  <div id="ratings">4.8<br> Raitings on Appstore</div>
+                  <div id="active-users">700&plus;<br> Active users</div>
+                </div>
+                </div>
+                <image width="100vw" height="150vh" src="./assets/images/first pic.png" alt="picture of a phone displaying ai ally"></image>
+            </div>
+            <div class="heading"><h1>Different destinations</h1></div>
+            <div class="second-paragraph">
+                <div>
+                    We explore the divrse ways in which this technology can revolutionize work across various industries and fields.
+                </div>
+                <div>
+                    Discover how AI can streamline processes and elevate your work to new heights
+                </div>
+                <div>Discover all applications &#8599;</div>
+            </div>
+            <div class="pics">
+                <div class="second-pic">
```

Comment:
It should be an image and a span tag with the text **Try is now**.
Use CSS to position the text in the container